runtime/lua/vim/shared.lua

```diff
@@ -174,6 +174,21 @@ function vim.tbl_filter(func, t)
   return rettab
 end
 
+--- Create a table with a given value repeated multiple times.
+---
+--- Does *not* copy the provided object, so the returned table contains
+--- identical objects.
+---
+---@param value the value to repeat
+---@param count integer the number of times to repeat the given value
+function vim.tbl_repeat(value, count)
```

Comment:
i would swap the arguments, because

1. the count is the most interesting part, since the idea is "i want to repeat this N times"
2. the count arg will usually be short, this avoids having it get hidden off-screen if `value` is a big expression

this is a pattern i generally recommend.
```suggestion
function vim.tbl_repeat(count, value)
```